x-pack/plugins/secops/public/pages/hosts/host_details.tsx

```diff
@@ -10,14 +10,26 @@ import {
   EuiPanel,
   EuiTitle,
 } from '@elastic/eui';
+import { getOr } from 'lodash/fp';
 import React from 'react';
 import { pure } from 'recompose';
 import styled from 'styled-components';
 import { getHostsUrl, HostComponentProps } from '../../components/link_to/redirect_to_hosts';
+import { UncommonProcessTable } from '../../components/page/hosts';
+import { AuthenticationTable } from '../../components/page/hosts/authentications_table';
+import { manageQuery } from '../../components/page/manage_query';
+import { AuthenticationsQuery } from '../../containers/authentications';
+import { GlobalTime } from '../../containers/global_time';
+import { UncommonProcessesQuery } from '../../containers/uncommon_processes';
 
-export const HostDetails = pure<HostComponentProps>(({ match }) => (
+const AuthenticationTableManage = manageQuery(AuthenticationTable);
+const UncommonProcessTableManage = manageQuery(UncommonProcessTable);
+
+const getFilterQuery = (hostId: string) => ({ term: { 'host.id': hostId } });
```

Comment:
Should this use the explicit `ESTermQuery` type?

Reply:
Yep. Good catch